### PR TITLE
Tts

### PR DESCRIPTION
**Pull request by Nikitak211**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const express = require('express');
 const path = require('path');
 const logger = require('./midware/logger')
 const txtgen = require('txtgen');
+const tts = require('./tts')
```
Comment (8o8inCodes) [inline app.js:5]: Delete this if you're not using it.

File: package.json [hunk 2]
```diff
@@ -18,10 +18,12 @@
   "homepage": "https://github.com/Nikitak211/textsnippet#readme",
   "dependencies": {
     "express": "^4.17.1",
+    "google-tts-api": "^2.0.2",
     "http": "0.0.1-security",
     "moment": "^2.29.1",
     "node": "^16.6.1",
     "nodemon": "^2.0.12",
+    "text-to-speech-js": "^1.1.10",
```
Comment (8o8inCodes) [inline package.json:26]: npm uninstall google-tts-api and text-to-speech-js please

File: public/script.js [hunk 3]
```diff
@@ -1,15 +1,24 @@
...
 button.addEventListener('click', getSnippet);
 
-const paragraph = document.getElementById('text-snippet');
-
 function getSnippet() {
     // call the server using "fetch" to '/generateText' API route
     // wait for it to return response (read about **promises** and **async await**), take the text that it provided
     // find the "text-snippet" element and set it's text to the response's text
+let speech = new SpeechSynthesisUtterance();
+speech.lang = "en";
```
Comment (8o8inCodes) [inline public/script.js:13]: Missing indentation

**Review by 8o8inCodes — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Nikitak211) [reply package.json:26]: ill do it at home need the pc for it

**Review by 8o8inCodes — APPROVED** [review]

**Comment by 8o8inCodes** [comment]
It's great! You can merge

**Pull request merged**